app/javascript/styles/rtl.scss

```diff
@@ -112,6 +112,16 @@ body.rtl {
     margin-right: 6px;
   }
 
+  .fa-ul {
+    margin-left: 0;
+    margin-left: 2.14285714em;
```

Comment:
2.14286132em might look a touch better.

Can you add the same for ar and he?

Thanks!